### PR TITLE
Add examples of 'failure'-type deployments

### PR DESCRIPTION
**Pull request by sebrandon1**
Working on adding examples of failures of each type of test suite with corresponding comments, etc.

Marked WIP as I add more manifests.

### CODE REVIEW

**Comment by sebrandon1** [comment]
Removed work in progress because I think I've added all of the examples I want to see.  I've left links and placeholders for all of the other tests listed in the catalog.

https://github.com/test-network-function/test-network-function/blob/main/CATALOG.md

**Review by hamadise — APPROVED** [review]

File: examples/accesscontrol/podRoleBinding.yaml [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+  creationTimestamp: null
+  labels:
+    run: rolebindingpod
+    app: test
+    test-network-function.com/generic: target
+    test-network-function.com/container: target
+  name: rolebindingpod
+  namespace: tnf
+spec:
+  # This should be set to false explicitly to satisfy testing requirements.
```
Comment (edcdavid) [inline examples/accesscontrol/podRoleBinding.yaml:45]: Is this =  "automountServiceAccountToken" ?

File: examples/accesscontrol/clusterRoleBinding.yaml [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+  creationTimestamp: null
+  labels:
+    run: clusterrolebindingpod
+    app: test
+    test-network-function.com/generic: target
+    test-network-function.com/container: target
+  name: clusterrolebindingpod
+  namespace: tnf
+spec:
+  # This should be set to false explicitly to satisfy testing requirements.
```
Comment (edcdavid) [inline examples/accesscontrol/clusterRoleBinding.yaml:46]: Same here, replace "this" with name

**Review by edcdavid — APPROVED** [review]

File: examples/accesscontrol/podRoleBinding.yaml (hunk 1 above)
Comment (sebrandon1) [reply examples/accesscontrol/podRoleBinding.yaml:45]: Good call, being explicit will make it clearer.  I'll adjust.

File: examples/accesscontrol/serviceAccountTokenPod.yaml [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+  labels:
+    run: serviceaccounttokenpod
+    app: test
+    test-network-function.com/generic: target
+    test-network-function.com/container: target
+  name: serviceaccounttokenpod
+  namespace: tnf
+spec:
+  # This should be set to false explicitly to satisfy testing requirements.
+  # Please visit this link to learn more:
```
[the remark this answers is not in the document]
Comment (sebrandon1) [reply examples/accesscontrol/serviceAccountTokenPod.yaml:15]: Sounds good, will fix.

**Pull request merged**